tera-templates/base.html

```diff
@@ -0,0 +1,43 @@
+{%- import "macros.html" as macros -%}
+{%- import "header/package_navigation.html" as navigation -%}
```

Comment:
This doesn't seem to be used at all?
```suggestion
```

Reply:
It is used, just in later stages

Reply:
Ok, please remove it until it is used. Otherwise I can't review it because I don't know the purpose - I don't have any context, and in the next PR I won't think to review it because it's already been merged.